bchrpc/server.go

```diff
@@ -670,8 +689,18 @@ func (s *GrpcServer) GetTransaction(ctx context.Context, req *pb.GetTransactionR
 			}
 		}
 
+		var tokenMetadata *pb.TokenMetadata
+		if req.IncludeTokenMetadata && tx.SlpTransactionInfo.ValidityJudgement == pb.SlpTransactionInfo_VALID {
+			tokenID, _ := chainhash.NewHash(tx.SlpTransactionInfo.TokenId)
```

Comment:
Always make sure to handle errors, some are not obvious like not enough memory to allocate chainhash. =)

Reply:
done